scripts/lib/common.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 CONTROLLER_TOOLS_VERSION="v0.7.0"
+HELM_VERSION="v3.9"
```

Comment:
Let's drop the version to `3.7` ? i guess 3.9 is not required.

Reply:
Or maybe we need to update our prow jobs to use 3.9 helm binaries..

Reply:
Helm is installed here https://github.com/aws-controllers-k8s/test-infra/blob/da4dad6c57818a0d4f87f8e2ac9e608612bfaff8/prow/jobs/images/Dockerfile.auto-generate-controllers and few other dockerfiles in the same directory

Reply:
Thanks for the review!
Which do you think is better? Do we drop the minimum required version to 3.7 for now? If so, I will modify the pull request.

Reply:
I'm thinking that maybe we should start by dropping the version to 3.7 and dealing with the upgrade in a different PR